cmd/beach/cmd/helpers.go

```diff
@@ -58,7 +58,12 @@ func readFileFromAssets(src string) string {
 }
 
 func getRelativePersistentResourcePathByHash(hash string) string {
-	return string(hash[0]) + "/" + string(hash[1]) + "/" + string(hash[2]) + "/" + string(hash[3])
+	slashPosition := strings.Index(hash, "/")
+	if slashPosition > 0 {
+		return string(hash[0:slashPosition]) + "/" + string(hash[slashPosition+1]) + "/" + string(hash[slashPosition+2]) + "/" + string(hash[slashPosition+3]) + "/" + string(hash[slashPosition+4])
```

Comment:
TBH, just ooked at this again and wonder… doesn't this return the first 5 characters of the hash? But… we use only 4? 🤔